docs/api/soil-based-model.rst

```diff
@@ -0,0 +1,71 @@
+.. currentmodule:: hydromt_wflow
+
+.. _api_wflowsbmmodel:
+
+=============
+WflowSbmModel
+=============
+
+The ``WflowSbmModel`` class represents the main hydrological model implementation.
+
+.. note::
+   Note that this class inherits from :class:`~hydromt_wflow.WflowBaseModel` and thereby includes all base functionalities, I/O routines, and setup methods from the base model.
+   Also, note that some setup methods are overridden or extended to include soil-based-specific parameters and data.
+   For a full list of inherited methods and attributes, see :ref:`WflowBaseModel <api_base_model>`.
+
+.. autosummary::
+   :toctree: ../_generated
+
+   WflowSbmModel
```

Comment:
This prints: Read or Write a wflow model.

I don't think this is really true though. We used to have Wflow SBM model class. Maybe we could then add a description under that says you can uses this to read, write, build, update wflow SBM models including config, staticmaps, forcing and states?
Also the arguments are printing werid for __init__. I think we should switch the parameters description from the class to the __init__ function like we did for the components (eg config).